Makefile

```diff
@@ -612,7 +616,7 @@ help:
 	@echo "   yetus          run Apache Yetus to check the quality of the source tree"
 	@echo
 	@echo "Commonly used build targets:"
-	@echo "   build-tools    builds linuxkit and manifest-tool utilities under build-tools/bin"
+	@echo "   build-tools    builds linuxkit utilities into under GOPATH/bin"
```

Comment:
yeah -- we really don't want them in GOPATH/bin

Reply:
That isn't quite so easy. We have to install it into `GOPATH/bin` as that is what `go get ...` does and it cannot be changed. We can change GOPATH, but then you lose all caching and redownload every time. So we will have to install it in `GOPATH/bin` and then copy it over.

Reply:
Actually, it isn't bad, I can play with the mounted directories. All good.

Reply:
Turned out to be a bit of a pain, because `go get` adds a subdirectory to the installed `bin/` when cross-compiling, but nothing a little extra step in the make target couldn't handle. Have another look.